### DISCUSSION TITLE
Schaltzeiten

### DISCUSSION THREAD

**Discussion by kht1011** [camueller/SmartApplianceEnabler#552] [Allgemein]
Hallo,

Ich habe eine Frage zu den Schaltzeiten. Ich habe ein Gerät, das täglich zwischen 6:00 und 20:00, 5 Stunden laufen soll, aber wenn genug PV- Strom da ist gerne auch länger Laufen darf. Im SAE habe ich das jetzt wie folgt eingetragen.

![image](https://github.com/camueller/SmartApplianceEnabler/assets/125148195/895150e8-224f-4ce4-89f8-0b33fc152e49)

Zur Zeit läuft das Gerät jetzt von kurz nach 6 Uhr bis kurz nach 19 Uhr, auch wenn der Strom dann bereits aus der Batterie stammt.
Das Ist gerade kein Problem, da die Batterie groß genug ist und genug PV Strom zu voll aufladen da ist, aber ich würde eventuelle Probleme mit der Einstellung gerne vor dem Winter lösen...

Gruß
Daniel

**Comment by camueller** [comment]
Der SAE ist nur Vemittler zwischen Deinem Gerät und dem SHM. Der SHM bestimmt, ob und wann das Gerät geschaltet wird. Wenn dabei Batterie oder ander Umstände nicht richtig berücksichtigt werden, kann Dir nur SMA helfen. Voraussichtlich werden sie das aber nicht tun, weil der SAE externe Software ist. 

Dein Schaltplan sieht ansonsten korrekt aus bezogen auf Deine Beschreibung.

**Comment by HerwigHahn** [comment]
Mein Anliegen ist ähnlich. Ich würde gerne die Wallbox  in der Nacht kurz nach Mitternacht aktivieren und dann (in der Zeit bis 6:00 Uhr morgens max. 20 kW laden. 
Ich habe u.s. Zeitpan erstellt, der aber nicht funktioniert. Was mir aufgefallen ist, dass ich bei mir nicht die Möglichkeit habe, eine Min/Max Laufzeit  anzugeben (felder fehlen). Liegt das daran? 
 ![grafik](https://github.com/camueller/SmartApplianceEnabler/assets/146930279/21905643-5f74-4095-816f-b18c36797c21)
